src/wallet/wallet2.cpp

```diff
@@ -4867,6 +5410,11 @@ std::vector<size_t> wallet2::select_available_outputs_from_histogram(uint64_t co
 //----------------------------------------------------------------------------------------------------
 uint64_t wallet2::get_num_rct_outputs()
 {
+  if(m_light_wallet) {
+    // TODO: openMonero doesnt have equivalent endpoint. Better solution? 
+    return 0;
+  }
```

Comment:
Not really needed: IIRC, this was a hack for just after rct was added, where the number of on-chain rct outs was very small and you couldn't get enough fake outs. This can be removed now I think.